src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -42,52 +52,59 @@ private static ArgumentParser getArgumentParser() {
                 .addMutuallyExclusiveGroup(PROGRAM_USAGE)
                 .required(false);
 
-        parser.addArgument("-h", "--help")
+        parser.addArgument(HELP_FLAGS)
+                .dest(HELP_FLAGS[0])
                 .help("Show help message.")
                 .action(new HelpArgumentAction());
 
-        mutexParser.addArgument("-config")
+        mutexParser.addArgument(CONFIG_FLAGS)
+                .dest(CONFIG_FLAGS[0])
                 .type(new ConfigFolderArgumentType())
                 .metavar("PATH")
                 .setDefault(EMPTY_PATH.toAbsolutePath())
-                .help("The directory containing the config files."
+                .help("The directory containing the config files. "
                         + "If not provided, the config files will be obtained from the current working directory.");
 
-        mutexParser.addArgument("-repo", "-repos")
+        mutexParser.addArgument(REPO_FLAGS)
                 .nargs("+")
-                .dest("repos")
+                .dest(REPO_FLAGS[0])
```

Comment:
doesn't this change the `repos` to `--repos`?

Reply:
Yes, it does. However, the way we get the result is also changed. 
`List<String> locations = results.get(REPO_FLAGS[0]);`
This is used to avoid the magic string.